esmvaltool/diag_scripts/monitor/multi_datasets.py

```diff
@@ -390,11 +439,13 @@ def __init__(self, config):
                 self.plots[plot_type].setdefault('x_pos_stats_avg', 0.0)
                 self.plots[plot_type].setdefault('x_pos_stats_bias', 0.92)
 
-            # Defaults for profile plots
-            if plot_type == 'profile':
+            # Defaults for zonal_mean_profile plots
+            if plot_type in ['zonal_mean_profile', 'profile', '1d_profile']:
                 self.plots[plot_type].setdefault('log_y', True)
```

Comment:
Don't forget to set a default for `log_x` (`False`) and `aspect_ratio` (`1.5`) for `1d_profile` here